ai-worker/Dockerfile

```
@@ -11,8 +11,23 @@ RUN pip install --no-cache-dir --prefix=/install .
 # ---------------------------------------------------------------------------
 FROM python:3.14-slim
 
-RUN groupadd --gid 1000 raven && \
-    useradd --uid 1000 --gid raven --create-home raven
+ARG TARGETARCH
+ARG DOTENVX_VERSION=1.61.0
+ARG DOTENVX_SHA256_AMD64=58fe4a5f84af835dce71e9e09bda47e7d54f0dc5b302bbf4f50224cc906646d3
+ARG DOTENVX_SHA256_ARM64=082a25127493197dcd9f167ba32986e2934af999c822fb5350ceda1da228d9ba
+RUN apt-get update && apt-get install -y --no-install-recommends curl \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)